### PR TITLE
Remove deploy-acm task

### PR DESCRIPTION
**Pull request by flaper87**
This is part of an effort to reduce the size of the code base so that we can focus on what's really the goal, to deploy edgeclusters

Signed-off-by: Flavio Percoco <flavio@redhat.com>

## Type of change

Please select the appropriate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] This change is a documentation update

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Comment by achuzhoy** [comment]
If you grep for capital ACM - more entries exist. Do you plan to have several MRs?

**Comment by flaper87** [comment]
> If you grep for capital ACM - more entries exist. Do you plan to have several MRs?

The remaining ones are for the `ACM_VERSION`  variable which I don't think we should touch for now :rofl: 

I do see a few others in the docs but we need to revamp the docs as far as the hub goes so I guess we will have to submit multiple PRs

File: pipelines/resources/deploy-ztp-hub.yaml [hunk 1]
```diff
@@ -81,9 +82,9 @@ spec:
         workspace: ztp
 
   # Deploy RHACM
```
Comment (achuzhoy) [inline pipelines/resources/deploy-ztp-hub.yaml:84]: change the comment too :)

**Comment by achuzhoy** [comment]
> > If you grep for capital ACM - more entries exist. Do you plan to have several MRs?
> 
> The remaining ones are for the `ACM_VERSION` variable which I don't think we should touch for now rofl
> 
> I do see a few others in the docs but we need to revamp the docs as far as the hub goes so I guess we will have to submit multiple PRs


do you want to update in pipelines/resources/hub/hub-deploy-icsp.yaml "This task will deploy RHACM on the Hub Cluster" ?

**Review by alknopfler — APPROVED** [review]
LGTM

**Pull request merged**